pkg/rke2/psp/psp.go

```diff
@@ -135,18 +135,18 @@ func kubeConfigExists(kc string) bool {
 // Load logic
 //
 // CIS:
-// - Check if the globalRestricted annotation does not exist, create PSP, role, binding.
-// - Check if the systemUnrestricted annotation does not exist, create the PSP, role, binding.
-// - Check if the globalUnrestricted annotation does not exist, check if PSP exists, and if so,
+// - If the globalRestricted annotation does not exist, create PSP, role, binding.
+// - If the systemUnrestricted annotation does not exist, create the PSP, role, binding.
+// - If the globalUnrestricted annotation does not exist, check if PSP exists, and if so,
 //   delete it, the role, and the bindings.
 //
 // non-CIS:
-// - Check if the globalUnrestricted annotation does not exist, create PSP, role, binding.
-// - Check if the systemUnrestricted annotation does not exist, create PSP, role, binding.
-// - Check if the globalRestricted annotation does not exist, check if the PSP exists and
-//   if it doesn't, create it. Check to see if the associated role and bindings exist and
+// - If the globalUnrestricted annotation does not exist, then create PSP, role, binding.
+// - If the systemUnrestricted annotation does not exist, then create PSP, role, binding.
+// - If the globalRestricted annotation does not exist, then check if the PSP exists and
+//   if it doesn't, create it. Check if the associated role and bindings exist and
 //   if they do, delete them.
-func SetPSPs(ctx *cli.Context, k8sWrapTransport transport.WrapperFunc) error {
+func SetPSPs(sCtx context.Context, ctx *cli.Context, k8sWrapTransport transport.WrapperFunc) error {
```

Comment:
why sCtx and not just ctx? There's a pretty strong convention for just calling the passed in context "ctx" and as far as this function is concerned, it knows nothing of the "s" part, which only the caller knows is related to signal handling.

Reply:
nice-to-have, but hopefully would be a simple replace

Reply:
Was not to trample the use of `ctx` throughout the repo as the ctx for the CLI args. I'm happy to switch to the more accepted convention for ctx and update all of the cli ctx refs.

Reply:
i see. fine as-is